runtime/syntax/man.vim

```diff
@@ -5,18 +5,21 @@ if exists('b:current_syntax')
   finish
 endif
 
-syntax case  ignore
-syntax match manReference      display '[^()[:space:]]\+([0-9nx][a-z]*)'
-syntax match manSectionHeading display '^\S.*$'
-syntax match manTitle          display '^\%1l.*$'
-syntax match manSubHeading     display '^ \{3\}\S.*$'
-syntax match manOptionDesc     display '^\s\+\%(+\|-\)\S\+'
+" Not contained because sometimes, manpages contain garbage backspaced
+" characters. See Restricted Shell at bash(1)
```

Comment:
Is this referring to the `+^Ho`?  It's actually not garbage!  I don't remember where I mentioned this, but the backspaces are for moving a print head back to stamp a character over the previous one.  `+^Ho` is a bullet point:

![dots](https://cloud.githubusercontent.com/assets/111942/21754706/765f6baa-d5d3-11e6-90a5-a5e0d00858e3.png)

There's a reference for this somewhere, but I can't remember the URL.